_dropin-templates/dropin-dictionary.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: DROPIN_NAME dictionary
+description: Learn how to customize the default dictionary for the DROPIN_NAME drop-in component.
+---
+
+import CodeInclude from '@components/CodeInclude.astro';
+import CodeImport from '@components/CodeImport.astro';
+import dictionary from '@dropins/storefront-DROPIN_PACKAGE/i18n/en_US.json.d.ts?raw';
+
+The default DROPIN_NAME dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the DROPIN_NAME drop-in component.
+
```

Comment:
```suggestion
The default DROPIN_NAME dictionary includes the default English values for all labels and text common to the DROPIN_NAME drop-in component.

```

Referencing an `i18n/en_US.json` file may cause confusion, as no such file exists in the Boilerplate, and developers do not have direct access to it. Instead, developers provide a JSON object to the drop-in’s initializer to override default values with custom labels or translations.